src/VisualizationTests/HelixImageComparisonTests.cs

```diff
@@ -465,22 +463,6 @@ public void WavySurfaceRender()
             RunCurrentModel();
             RenderCurrentViewAndCompare(MethodBase.GetCurrentMethod().Name);
         }
-        #endregion
-
-        #region SpecialRenderPackages
-        [Test]
-        public void DirectManipulator()
```

Comment:
can you also remove the reference images?

Reply:
done